components/Login.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="login-wrapper border border-light">
+    <form
+      class="form-signin"
+      @submit.prevent="login">
+      <h2 class="form-signin-heading">Please sign in</h2>
+      <label
+        for="inputUsername"
+        class="sr-only">Username</label>
+      <input
+        v-model="username"
+        type="text"
+        id="inputUsername"
+        class="form-control"
+        placeholder="Username"
+        required
+        autofocus>
+      <label
+        for="inputPassword"
+        class="sr-only">Password</label>
+      <input
+        v-model="password"
+        type="password"
+        id="inputPassword"
+        class="form-control"
+        placeholder="Password"
+        required>
+      <button
+        class="btn btn-lg btn-primary btn-block"
+        type="submit">Sign in</button>
+    </form>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+import qs from "qs";
+
+// const apiUrl = process.env.API_URL;
+
+// @TODO: we won't need this when the app is landoified.
+if (process.env.NODE_ENV !== "production") {
+  let dotenv = require("dotenv");
+  dotenv.load();
+}
+
+export default {
+  name: "Login",
+  data() {
+    return {
+      username: "",
+      password: ""
+    };
+  },
+  methods: {
+    login() {
+      const data = {
+        username: this.username,
+        password: this.password,
+        grant_type: "password",
+        client_id: "ee03f762-aace-438e-8724-01d82134d8ce",
```

Comment:
Should these be env vars?
```suggestion
        client_id: process.env.API_CLIENT_ID,
```